recipe/meta.yaml

```
@@ -135,6 +135,13 @@ outputs:
         - lib/pkgconfig  # [unix]
         - lib/libmagma_sparse.so  # [linux]
         - Library\bin\magma_sparse.dll  # [win]
+    build:
+      ignore_run_exports:
+        - cuda-version
+      ignore_run_exports_from:
```

Personally, I'd add an explanatory comment here, à la
```suggestion
      # Ignore run exports to implement CUDA minor version compatibility
      ignore_run_exports_from:
```